spec/features/admin/authentication_spec.rb

```diff
@@ -9,17 +9,14 @@
   let!(:enterprise) { create(:enterprise, owner: user) } # Required for access to admin
 
   scenario "logging into admin redirects home, then back to admin" do
-    # This is the first admin spec, so give a little extra load time for slow systems
-    Capybara.using_wait_time(120) do
```

Comment:
:see_no_evil:

Reply:
:rofl:

Reply:
I remember that one! That was tricky. Somehow this spec would always fail on CI but work locally. Finally we found that it was the first spec running and that the first spec always need a while to boot up...